OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -206,11 +215,36 @@ DampedFiberVelocityCalculationResult calcDampedNormFiberVelocity(
 
     return {
         dlceN_dt,
+        fv,
         err,
         converged,
     };
 }
 
+/* Calculates the stiffness of the fiber in the direction of the fiber.
```

Comment:
I can't see where the original implementation was removed (it's removed in the header?)

Reply:
Original implementation was removed in `Millard2012EquilibriumMuscle.cpp` lines 1326 and in header on line 589